src/CommunityVoices/Public/js/batch-upload-confirm.js

```diff
@@ -0,0 +1,329 @@
+toRemove = []; // after we upload form content, we want to delete all qutotes/sources at the same time
+
+const fieldsToChange=[["originalquote","originalText"],["editedquotes","text"],["intervieworfileaccessdate","dateRecorded"],["interviewerresearcher",'interviewer'],["subattribution","subAttribution"],["quotationmarks","quotationMarks"]]; // add to later
```

Comment:
Is "add to later" relevant?

Reply:
@Sammidysam I can probably get rid of this. I am already giving users feedback when the quote upload will fail, (i.e. request will return code besides 200). If for some reason the request fails for some other reason, I don't think the user needs to know this.

Reply:
I was more responding to comment at the end of the line here.  I was not at all trying to question this variable's existence, but rather just if the comment was still relevant at this point in the life of the code.  I do not know.

Reply:
Removed comment